code/modules/mob/living/carbon/human/human.dm

```diff
@@ -375,14 +375,14 @@
 /mob/living/carbon/human/can_inject(mob/user, target_zone, injection_flags)
 	. = TRUE // Default to returning true.
 	if(user && !target_zone)
-		target_zone = user.zone_selected
+		target_zone = check_zone(user.zone_selected)
 	// we may choose to ignore species trait pierce immunity in case we still want to check skellies for thick clothing without insta failing them (wounds)
 	if(injection_flags & INJECT_CHECK_IGNORE_SPECIES)
 		if(HAS_TRAIT_NOT_FROM(src, TRAIT_PIERCEIMMUNE, SPECIES_TRAIT))
 			. = FALSE
 	else if(HAS_TRAIT(src, TRAIT_PIERCEIMMUNE))
 		. = FALSE
-	var/obj/item/bodypart/the_part = isbodypart(target_zone) ? target_zone : (get_bodypart(target_zone) || get_bodypart(BODY_ZONE_CHEST))
+	var/obj/item/bodypart/the_part = isbodypart(target_zone) ? target_zone : get_bodypart(check_zone(target_zone))
```

Comment:
get_bodypart(BODY_ZONE_CHEST) was redundant since check_zone(target_zone) will return the chest if passed a null value.